conformance-test/testBenchUtil.ts

```diff
@@ -0,0 +1,34 @@
+/*!
```

Comment:
nit: I would be tempted to cll this `test-bench-util.ts`, this just tends to be the JavaScript convention.